python_modules/dagster-graphql/dagster_graphql/schema/asset_graph.py

```diff
@@ -76,6 +76,16 @@ def resolve_asset(self, _graphene_info):
         )
 
 
+class GrapheneAssetLiveInfo(graphene.ObjectType):
```

Comment:
maybe `AssetLatestInfo` instead of `AssetLiveInfo`?

Reply:
changed! ✅